Hassan_Ali/tree.py

```diff
@@ -0,0 +1,52 @@
+class TreeNode:
+    def __init__(self,value,weight):
+        self.value = value
+        self.left = None
+        self.right = None
+        self.weight = weight
+# printing using inorder
+def print_tree(tree):
+    if not tree:
+        return
+
+    print(tree.value,tree.weight)
+    print_tree(tree.left)
+    print_tree(tree.right)
+
+def find_path(root,city):
```

Comment:
Can you create another function for finding the distance, instead of having both finding the distance and the path in one function. 
Also, can you annotate your functions. Example: 
`def find_path(root: TreeNode, city: str) -> str: `

`def find_distance(root: TreeNode, city: str) -> int:`

Reply:
I already created it